app/src/main/java/me/ash/reader/domain/service/GoogleReaderRssService.kt

```diff
@@ -228,7 +228,7 @@ class GoogleReaderRssService @Inject constructor(
 
                 // 2. Fetch folder and subscription list
                 googleReaderAPI.getSubscriptionList()
-                    .subscriptions.groupBy { it.categories?.first() }
+                    .subscriptions.sortedBy { it.categories?.sortedBy { it.label }?.first()?.label }.groupBy { it.categories?.first() }
```

Comment:
```suggestion
                    .subscriptions.groupBy { it.categories?.first() }
                    .toSortedMap { c1, c2 -> c1?.label.toString().compareTo(c2?.label.toString()) }
```

Reply:
@JunkFood02 Yeah, I like this better. Tested it and it works well. Added!